### PR TITLE
T32209 kolibri launcher

### PR DESCRIPTION
**Pull request by danigm**
https://phabricator.endlessm.com/T32209

### CODE REVIEW

File: KolibriShortcut/KolibriShortcut.vcxproj [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>16.0</VCProjectVersion>
+    <Keyword>Win32Proj</Keyword>
+    <ProjectGuid>{5A0416CD-8F36-4F72-9A42-A9FF5F0823B6}</ProjectGuid>
+    <RootNamespace>KolibriShortcut</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <ProjectName>Endless Key Launcher</ProjectName>
```
Comment (erikos) [inline KolibriShortcut/KolibriShortcut.vcxproj:19]: Is that the user visible name of the app?

**Review by erikos — COMMENTED** [review]
Looks good overall, though Dylan knows this part better of course.

File: KolibriShortcut/KolibriShortcut.vcxproj (hunk 1 above)
Comment (danigm) [reply KolibriShortcut/KolibriShortcut.vcxproj:19]: Yes, that's the final binary name, you can see the exe file generated in the artifact here: https://github.com/endlessm/windows-usb-launcher/suites/3187982814/artifacts/73564029

Comment (erikos) [reply KolibriShortcut/KolibriShortcut.vcxproj:19]: Perfect, thanks.

**Pull request merged**